assets/src/components/v2/bus_shelter/normal_body.tsx

```diff
@@ -1,25 +1,78 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 
 import Widget, { WidgetData } from "Components/v2/widget";
+import makePersistent from "./persistentHOC";
+import useInterval from "Hooks/use_interval";
+import FlexZonePageIndicator from "Components/v2/flex/page_indicator";
 
 interface Props {
   main_content: WidgetData;
   flex_zone: WidgetData;
   footer: WidgetData;
 }
 
+const ProofOfConceptElevatorWidget = ({ data, lastUpdate, onFinish }) => {
```

Comment:
Might be a good idea to put this file back to normal now that the ElevatorStatus widget skeleton has been started. Either that or move it to PreFare because BusShelter wouldn't use the widget anyway.

Reply:
Mmm definitely, will shift it to PreFare